src/qutip_qip/circuit.py

```diff
@@ -2056,6 +2057,11 @@ def latex_code(self):
     def _raw_png(self):
         return _latex.image_from_latex(self.latex_code(), "png")
 
+    def _raw_img(self, file_type="png", density_dpi=100):
+        return _latex.image_from_latex(
+            self.latex_code(), file_type, density_dpi
+        )
+
     if "png" in _latex.CONVERTERS:
         _repr_png_ = _raw_png
 
```

Comment:
I guess we can now safely remove `_raw_png` and `_raw_svg` with your `_raw_img`.
```python
    def _raw_img(self, file_type="png", density_dpi=100):
        return _latex.image_from_latex(
            self.latex_code(), file_type, density_dpi
        )

    if "png" in _latex.CONVERTERS:
        _repr_png_ = _raw_img

    if "svg" in _latex.CONVERTERS:
        _repr_svg_ = partial(_raw_img, file_type="svg", density_dpi=None)
```